File: 0001 Code Chef/0021 Biryani classes/implementation/0010 nodejs/biryani_classes.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    let test = parseInt(input);
    let count = 0;
    rl.on('line',(input)=>{
        string_array = input.split(" ");
        x = parseInt(string_array[0]);
        y = parseInt(string_array[1]);
        total = x * y;
        console.log(total);
        count = count + 1;
        if(count == test){
            rl.close();
        }
    })
})

File: 0001 Code Chef/0021 Biryani classes/implementation/0008 python3/biryani_classes.py
if __name__ == "__main__":
    test = int(input())
    while(test >0):
        string_input = input()
        string_array = string_input.split(" ")
        x = int(string_array[0]);
        y = int(string_array[1]);
        total = x * y
        print(total)
        test = test - 1;